gnssanalysis/gn_download.py

```diff
@@ -503,27 +503,27 @@ def attempt_ftps_download(
 
 
 def attempt_url_download(
-    download_dir: _Path, url: str, filename: str = None, type_of_file: str = None, if_file_present: str = "prompt_user"
+    download_dir: _Path, url: str, download_filename: str = None, type_of_file: str = None, if_file_present: str = "prompt_user"
```

Comment:
If we are changing `download_filename` here, we'll need to make sure all functions that call this, use the new argument name

Reply:
Changed back to `filename` as it might affect other external scripts including `auto_download_PPP.py`